styles/vendor/fotorama/modules/_thumb.scss

```diff
@@ -0,0 +1,66 @@
+.fotorama__thumb {
+    position: relative;
+    overflow: hidden;
+    width: 100%;
+    height: 100%;
+    background-color: $gray-lighter;
+
+    .fotorama__nav__frame:focus & {
+        box-shadow: none;
+        z-index: 2;
+        &:after {
+            position: absolute;
+            top: 0;
+            bottom: 0;
+            left: 0;
+            right: 0;
+            box-shadow: $focus__box-shadow;
+            border-radius: inherit;
+            content: '';
+            z-index: $z-index-10;
+        }
+    }
+
+    .fotorama__nav__frame--thumb:focus &,
+    .fotorama__nav__frame--dot:focus & {
+        overflow: inherit;
+    }
+
+    .fotorama__nav__frame:nth-child(2):focus &:after {
+        left: 1px;
+    }
+
+    .fotorama__nav__frame:last-child:focus &:after {
+        right: 1px;
+    }
+
+    &--icon {
+        @extend .fotorama-sprite;
+        padding-bottom: $fotorama-thumb-arrow;
+        font-size: 0.001px;
```

Comment:
I don't get what you want to achive here.
If you want to add empty text with all properties, maybe just add `&nbsp`?